web/src/main/webapp/xslt/services/inspire-atom/opensearch.xsl

```diff
@@ -78,7 +78,7 @@
           <Url type="text/html" rel="results">
             <xsl:attribute name="template">
               <xsl:value-of
-                select="concat($baseUrl,/root/gui/url,'/opensearch/', /root/gui/language, '/catalog.search#/search?any={searchTerms?}')"/>
+                select="concat($baseUrl,/root/gui/url,'/search?any={searchTerms?}')"/>
```

Comment:
The url created now is http://localhost:8080/geonetwork/search?any={searchTerms?}, that service doesn't exist.

The purpose of this link is to display the html search results?

To show the standard search results page, the url format should be:

- http://localhost:8080/geonetwork/srv/eng/catalog.search#/search?any={searchTerms?}

I see it's also configured this urlrewrite rule:

- http://localhost:8080/geonetwork/srv/eng/opensearch/htmlsearch?q=QUERYVALUE --> 
http://localhost:8080/geonetwork/srv/eng/catalog.search#/search?any=QUERYVALUE

Any of the formats should work, please update the PR to fix it.